### DISCUSSION TITLE
Why？ Exception while fetching data (/analyze) : null

### DISCUSSION THREAD

**Discussion by cqxing** [apache/skywalking#12083] [Q&A]
When using the **Trace Profiling** function, if the data is successfully collected, **click on Analyze** and an error will appear

query：
{"query":"query getProfileAnalyze($queries: [SegmentProfileAnalyzeQuery!]!) {\n  analyze: getSegmentsProfileAnalyze(queries: $queries) {\n    tip\n    trees {\n      elements {\n        id\n        parentId\n        codeSignature\n        duration\n        durationChildExcluded\n        count\n      }\n    }\n  }\n  }","variables":{"queries":[{"segmentId":"51abcf8fc8f44aaca09137db35b51b2d.76.17127319823107230","timeRange":{"start":1712731982311,"end":1712731982314}}]}}

response：
{
    "errors": [
        {
            "message": "Exception while fetching data (/analyze) : null",
            "locations": [
                {
                    "line": 2,
                    "column": 3
                }
            ],
            "path": [
                "analyze"
            ],
            "extensions": {
                "classification": "DataFetchingException"
            }
        }
    ],
    "data": null
}

**Comment by cqxing** [comment]
apache-skywalking-apm-9.5.0

**Comment by wu-sheng** [comment]
1. Try the latest version
2. Help yourself first. If you don't even check the logs and debug, I don't think we can help you.
